File: Week02/Day1/draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

lines = int(input("How many lines should I draw? "))

def draw_triangle(lines):
    a = 1
    while a <= lines:
        print("*" * a)
        a += 1

draw_triangle(lines)

File: Week07/Day1/draw_triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

for (let i = 1; i <= lineCount; i++) {
    let str = ''
    for (let j = i; j = i + 1; j++){
        str += '*'
    }
    console.log(str)
}


